org.videolan.VLC.json

```diff
@@ -887,6 +887,24 @@
         }
       ]
     },
+    {
+      "name": "smbclient",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./buildtools/bin/waf configure --prefix=/app --disable-python --without-ad-dc --without-acl-support --without-ldap --without-ads --without-pam --without-systemd",
+        "./buildtools/bin/waf build -j `nproc` --targets=smbclient",
+        "mkdir -p /app/lib",
+        "find bin -type f -name '*.so' -exec cp {} /app/lib \\;",
+        "install -D bin/default/include/public/libsmbclient.h /app/include/libsmbclient.h",
+        "install -D bin/default/source3/libsmb/smbclient.pc /app/lib/pkgconfig/smbclient.pc"            ],
```

Comment:
`]` should be on newline, `sources` at wrong indentation level.

Reply:
mode for header and pc file should be 644.

Reply:
Surely you could do `install --target-directory=/app/lib bin/*.so` rather than that find mess (and mkdir)

Reply:
I can't just use wildcards, because the *.so are all scattered around in that directoy. And "waf install" seems to be broken for that target/configuration (need to file an upstream bug).

About mode for .h/.pc: These are deleted in the cleanup phase anyway?!

Reply:
> I can't just use wildcards, because the *.so are all scattered around in that directoy. And "waf install" seems to be broken

Alright.

> About mode for .h/.pc: These are deleted in the cleanup phase anyway?!

True but there is no reason to make them executable anyway.